src/main/proto/ReportService.proto

```diff
@@ -149,20 +149,29 @@ message JiraConfigsRequest {
     JiraParameters jiraParameter = 1;
 }
 
-message createjiraTicketRequest{
+message CreateJiraTicketRequest {
     int32 id =1;
 }
 
-message createjiraTicketResponse{
+message CreateJiraTicketResponse {
     string message = 1;
     string jiraTicketLink = 2;
 }
 
-message getAllJiraRequest{
+message GetAllJiraRequest {
     string status = 1;
+    int32 pageSize = 2;
+    int32 page = 3; 
+}
+message GetAllJiraResponse {
+    int32 totalRecords = 1;
+    repeated JiraTickets records = 2;
```

Comment:
shouldn't `JiraTickets` be in singular as `JiraTicket`? I believe the message `JiraTickets` represents only one ticket, isn't it?

Reply:
no, its list of created jira